src/basics/installextensions.md

```diff
@@ -41,7 +41,7 @@ Führen Sie die folgenden Schritte aus, um diese Erweiterung zu installieren:
 </xmp></pre>
 1. Da Sie root-Rechte benötigen, um in das lib-Verzeichnis schreiben zu können,
    muss der folgende Befehl zum Entpacken des Archivs mit sudo ausgeführt werden:
-<pre><xmp style="margin:0; font-size: .85em;">sudo unzip /home/user/Downloads/geoserver-{{ book.geoServerVersion }}-wps-plugin.zip
+<pre><xmp style="margin:0; font-size: .85em;">sudo unzip /home/user/Downloads/geoserver-2.13.2-wps-plugin.zip
```

Comment:
Is the variable not working here (and above)? It should be `2.15.1` either way.

Reply:
The [link](https://raw.githubusercontent.com/terrestris/geoserver-in-action-ws/master/src/materials/geoserver-2.13.2-wps-plugin.zip) to download this zip-file (geoserver-2.13.2-wps-plugin.zip) only works with `2.13.2` and not with `2.15.1`. The variable *book.geoServerVersion* would change the download link to https://raw.githubusercontent.com/terrestris/geoserver-in-action-ws/master/src/materials/geoserver-2.15.1-wps-plugin.zip which does not work.